src/main/scala/edu/tulane/cs/hetml/nlp/sprl/VisualTriplets/VisualTripletsApp.scala

```diff
@@ -16,37 +16,31 @@ import scala.collection.mutable.ListBuffer
   */
 object VisualTripletsApp extends App {
 
-  val flickerTripletReader = new ImageTripletReader("data/mSprl/saiapr_tc-12/imageTriplets", "Flickr30k.majorityhead")
-  val msCocoTripletReader = new ImageTripletReader("data/mSprl/saiapr_tc-12/imageTriplets", "MSCOCO.originalterm")
-  val isTrain = false
+//  val flickerTripletReader = new ImageTripletReader("data/mSprl/saiapr_tc-12/imageTriplets", "Flickr30k.majorityhead")
+//  val msCocoTripletReader = new ImageTripletReader("data/mSprl/saiapr_tc-12/imageTriplets", "MSCOCO.originalterm")
+//  val externalTrainTriplets = flickerTripletReader.trainImageTriplets //msCocoTripletReader.trainImageTriplets   ++
+//  val testTriplets = flickerTripletReader.testImageTriplets //flickerTripletReader.testImageTriplets ++
+
+  val frequentPrepositions = List("in", "on", "between", "in front of", "behind", "above", "in between", "around", "over")
+
+  val isTrain = true
   val classifierDirectory = s"models/mSpRL/VisualTriplets/"
-  val classifierSuffix = "combined_perceptron"
-  val classifierOnSuffix = "combined_on_perceptron"
-  val classifierInSuffix = "combined_in_perceptron"
-  val classifierAboveSuffix = "combined_above_perceptron"
-  val classifierInFrontOfSuffix = "combined_InFrontOf_perceptron"
-  val externalTrainTriplets = flickerTripletReader.trainImageTriplets ++ msCocoTripletReader.trainImageTriplets
-  val testTriplets = flickerTripletReader.testImageTriplets ++ msCocoTripletReader.testImageTriplets
+  val classifierSuffix = "vgg_perceptron"
 
-//  val visualClassifier = new VisualTripletClassifiers.VisualTripletClassifier()
 
-//  visualClassifier.modelSuffix = classifierSuffix
-//  visualClassifier.modelDir = classifierDirectory
+  if (isTrain) {
 
-  VisualTripletOnClassifier.modelSuffix = classifierOnSuffix
-  VisualTripletOnClassifier.modelDir = classifierDirectory
+    val VGGJsonReader = new JSONReader()
+    VGGJsonReader.readJsonFile("data/mSprl/saiapr_tc-12/VggData/")
 
-  VisualTripletInClassifier.modelSuffix = classifierInSuffix
-  VisualTripletInClassifier.modelDir = classifierDirectory
+    val vggTriplets = VGGJsonReader.trainImageTriplets.take(2000)
```

Comment:
please check the papers, are there any standard/commonly used train/test split for VG data?